imagetagger/imagetagger/annotations/views.py

```diff
@@ -234,11 +234,11 @@ def export_format(export_format_name, imageset):
 
     placeholders_filename = {
         '%%imageset': imageset.name,
-        '%%team': imageset.team.name,
+        '%%team': imageset.team.name if imageset.team else ''
```

Comment:
shouldn't there be a "," at the end of the line?

Reply:
Yes, you are right, it is fixed now.